docs/user-documentation/metadata.md

```diff
@@ -83,11 +83,35 @@ Each of these vocabularies has its own set of fields allowing repositories to fu
 
 The vocabularies provided by default are a starting point, and a repository administrator can create whatever vocabularies are desired.
 
-## Field Types
+## Field Data Types
```

Comment:
I think this section is about Drupal's field types. I find 'Field Data Types' to be ambiguous. It might refer to the data type of the value as stored in the database, which is different from (and hidden, to an extent by) the field type.

Reply:
That makes sense, I was just going off what Drupal called them. Happy to keep as is